packages/jellyfish-api-core/__tests__/category/account/getAccountHistory.test.ts

```diff
@@ -0,0 +1,84 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing } from '@defichain/jellyfish-testing'
+import waitForExpect from 'wait-for-expect'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+
+function createTokenForContainer (container: MasterNodeRegTestContainer) {
+  return async (address: string, symbol: string, amount: number) => {
+    const metadata = {
+      symbol,
+      name: symbol,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: address
+    }
+    await container.call('createtoken', [metadata])
+    await container.generate(1)
+
+    await container.call('minttokens', [`${amount.toString()}@${symbol}`])
+    await container.generate(1)
+  }
+}
+
+describe('Account', () => {
+  const testing = Testing.create(new MasterNodeRegTestContainer())
+  const createToken = createTokenForContainer(testing.container)
+
+  beforeAll(async () => {
+    await testing.container.start()
+    await testing.container.waitForWalletCoinbaseMaturity()
+    await testing.container.waitForWalletBalanceGTE(100)
+    await createToken(await testing.container.getNewAddress(), 'DBTC', 200)
```

Comment:
since we are now using testing instance for this how about we utilize the methods available there?
```
const metadata = {
      symbol: 'DBTC',
      name: 'DBTC',
      isDAT: true,
      mintable: true,
      tradeable: true,
      collateralAddress: await testing.container.getNewAddress()
    }
    await testing.token.create(metadata)
    await testing.container.generate(1)
    await testing.token.mint({ amount: 200, symbol: 'DBTC' })
    await testing.container.generate(1)
```